cpp/src/sort/sort.cu

```diff
@@ -37,7 +38,7 @@ std::unique_ptr<column> sorted_order(table_view const& input,
                                      rmm::cuda_stream_view stream,
                                      rmm::mr::device_memory_resource* mr)
 {
-  return sorted_order<false>(input, column_order, null_precedence, stream, mr);
+  return sorted_order<sort_method::UNSTABLE>(input, column_order, null_precedence, stream, mr);
```

Comment:
While I was in the area, moving to consistently using the enum-based approach, since it is slightly more self-documenting at the call site.